src/evaluation/WorkspaceManager.jl

```diff
@@ -97,6 +97,9 @@ function resolved_environment_path(notebook::Notebook)
     else
         if isabspath(notebook.environment_path)
             tamepath(notebook.environment_path)
+        elseif notebook.environment_path == "@."
```

Comment:
this fix the issue that when notebooks are specified with `@.` the project is using `notebook_dir/@.` path.